elements/pfe-dropdown/src/pfe-dropdown.scss

```diff
@@ -0,0 +1,160 @@
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: dropdown;
+
+$variables: (
+  toggle: (
```

Comment:
Totally an optional thing, but I just wanted to plant the idea, for now or the future. :) 

It might be easier to keep the variables map with most properties set at the root level, and then nest the handful in a section if needed, i.e.

```scss
$variables: (
  MinWidth: calc(#{pfe-var(container-spacer)} * 2),
  MaxWidth: 100%,
  ...
  BorderLeftColor--before: #ededed,
  BorderStyle--before: solid,

  menu: (
    SpacingVertical: calc(#{pfe-var(container-spacer)} * 0.5),
    SpacingHorizontal: 0,
    ...
    MinWidth: 100%
  )
);
```

That way when you want to reference them, you don't have to include the $region very often
`min-width: #{pfe-local($cssvar: MinWidth, $region: toggle)};`
becomes
`min-width: #{pfe-local( MinWidth )};`